addons/skin.estuary/xml/DialogVideoInfo.xml

```diff
@@ -434,22 +434,22 @@
 							<control type="image">
 								<top>20</top>
 								<left>20</left>
-								<width>204</width>
-								<height>277</height>
+								<width>186</width>
+								<height>279</height>
 								<texture background="true">$INFO[ListItem.Art(poster)]</texture>
-								<aspectratio aligny="center">keep</aspectratio>
+								<aspectratio aligny="center">scale</aspectratio>
```

Comment:
Is scaling going to be a problem re cropping if poster art isn't in this 0.667 ratio?

If it were just me, I'd have no issue with scale, but does differ from fanart.tv poster sizing which is 0.701 (1000/1426)
https://wiki.fanart.tv/ImageTypes/Movies/movieposter/

Always got the sense that Estuary preferred "keep" to deal with those sizing inconsistencies but not quite sure what to do here where the layout design clearly calls for "scale" to have artwork line up correctly.

Reply:
Can the overlay be a bordertexture of the poster? Because that should allow `keep` as aspectratio.
Then add conditional visibility for DefaultVideo image to only show on `String.IsEmpty(ListItem.Art(poster))`

Reply:
Actually looking at the screenshot further, what is the benefit of the overlay when the poster is visible? Unless this is a specific design choice for consistency, why not simply remove it completely if the item has a poster?

Reply:
You're right, they don't serve any purpose here. I think they may just be inherited from the actor thumbs code where it's needed for the name/role labels. I'll remove them altogether.

Reply:
Actually they're still needed when there's no poster available for the textbox so I'll make them conditional.

As for the `scale` vs `keep` issue - `keep` once again messes with background image and the focus border:

![screenshot00013](https://github.com/xbmc/xbmc/assets/133808/74d99865-6139-49de-904f-4ec1c9832d1c)

Reply:
Any ideas?

Reply:
Can focus border be in `<bordertexture>` for poster? Because that should then adapt to size.

Reply:
It works for the posters but not the DefaultMovies.png and border behind them.

![screenshot00014](https://github.com/xbmc/xbmc/assets/133808/8bbc2810-d836-42e4-8708-7ce8e5874980)

And there's no focus when there's no poster.

![screenshot00015](https://github.com/xbmc/xbmc/assets/133808/9a321211-7a1d-4c08-b240-e5ab86e8b46c)

Reply:
I think I've sussed it.

![screenshot00017](https://github.com/xbmc/xbmc/assets/133808/6102dd92-e23a-4b81-a3dd-7fc0c48bec4b)

![screenshot00018](https://github.com/xbmc/xbmc/assets/133808/d4257ec0-4e17-4819-98b7-a8b6f2e61aab)

![screenshot00019](https://github.com/xbmc/xbmc/assets/133808/6bbaa660-1f34-4523-b6de-22111606b708)

Reply:
> Always got the sense that Estuary preferred "keep" to deal with those sizing inconsistencies but not quite sure what to do here where the layout design clearly calls for "scale" to have artwork line up correctly.

Actually I just checked and Estuary uses scale for other views.

Reply:
> You're right, they don't serve any purpose here. I think they may just be inherited from the actor thumbs code where they're needed for the name/role labels. I'll remove them altogether.

@jjd-uk

Reply:
Ah right. So did you not get to removing in the PR?

Reply:
> Actually I just checked and Estuary uses scale for other views.

Ah, you're right. I must've mixed it up because of that discussion a while back about Shift view needing keep.

In that case, maybe it should be scale for the sake of consistency?

Sorry for the wild goose chase! Removing the overlay is still a good improvement, so not all for nothing.